src/bounds.test.ts

```diff
@@ -19,7 +19,7 @@ describe("lowerBound", () => {
 
 describe("upperBound", () => {
   test.each([
-    ["^16.8.0", "<17.0.0"],
+    ["^16.8.0", "<17.0.0-0"],
```

Comment:
This change originates here:
https://github.com/mitchellhamilton/sembear/blob/d4eafaaebd3629a4f737e47d04cf307e1e1d449d/src/bounds.ts#L20
The created range already has that prerelease included there.

I've "bisected" this and it seems that this has been changed in `semver@7.3.1`, so somewhere here:
https://github.com/npm/node-semver/compare/v7.3.0...v7.3.1